SwitchSpeak-iOS/SwitchSpeak-iOS/Settings/UserSettings.swift

```diff
@@ -58,5 +67,19 @@ struct UserSettings : Codable {
         gridSize = .MEDIUM
         shouldAlert = true
         vocabLevel = .SLIGHTLY_MORE
+        voiceType = .MALE_VOICE
```

Comment:
I only made this male because the iOS default is female and I wanted to test the different voice.